src/kernel/src/fs/dev/dmem1.rs

```diff
@@ -24,12 +28,20 @@ impl VFileOps for Dmem1 {
     fn ioctl(
         &self,
         _: &crate::fs::VFile,
-        _: u64,
-        _: &mut [u8],
+        com: u64,
+        mut data: &mut [u8],
         _: &Ucred,
         _: &VThread,
     ) -> Result<(), Box<dyn Errno>> {
-        todo!()
+        match com {
```

Comment:
Why skip the 3 checks before this?

Reply:
I forgot because I was too excited :( I'm looking at this now and wondering. On the PS4, the auth field of ucred starts at offset 0x58, right? Otherwise, nothing makes sense to me.

Reply:
Correct.